examples/2-handle-prompt/package.json

```
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "node index.js",
+    "watch": "node --watch index.js"
+  },
+  "dependencies": {
+    "@copilot-extensions/preview-sdk": "latest",
+    "@octokit": "latest"
```

Don't use `latest` for external dependencies.

```suggestion
    "octokit": "^4.0.2"
```